draft-packages/menu/docs/Menu.stories.tsx

```diff
@@ -97,328 +32,143 @@ export default {
   decorators: [withDesign],
 }
 
-export const LabelAndIcon = () => (
-  <Menu
-    button={<Button label="Label" icon={chevronDown} iconPosition="end" />}
-    dropdownId="dropdown"
-  >
-    <MenuInstance />
-  </Menu>
-)
-
-LabelAndIcon.storyName = "Label and Icon (Kaizen Site Demo)"
-
-export const AutoHideBehaviours = () => (
-  <>
-    <Box mb={1}>
-      <Paragraph variant="body">
-        Auto hide turned on (default behaviour):
-      </Paragraph>
-      <Menu
-        button={<Button label="Label" icon={chevronDown} iconPosition="end" />}
-        dropdownId="dropdown"
-        autoHide="on"
-      >
-        <MenuInstance />
-      </Menu>
-    </Box>
-    <Box mb={1}>
-      <Paragraph variant="body">
-        Auto hide when clicking outside the menu, but not inside:
-      </Paragraph>
-      <Menu
-        button={<Button label="Label" icon={chevronDown} iconPosition="end" />}
-        dropdownId="dropdown"
-        autoHide="outside-click-only"
-      >
-        <MenuInstance />
-      </Menu>
-    </Box>
-    <Box mb={1}>
-      <Paragraph variant="body">Auto hide turned off:</Paragraph>
-      <Menu
-        button={<Button label="Label" icon={chevronDown} iconPosition="end" />}
-        dropdownId="dropdown"
-        autoHide="off"
-      >
-        <MenuInstance />
-      </Menu>
-    </Box>
-  </>
-)
-
-AutoHideBehaviours.storyName = "Auto hide behaviours"
-
-export const LabelAndIconReversed = () => (
+export const DefaultStory = args => (
```

Comment:
I'd assume many people would look at the default story for a code example, but its menu content is very simplistic - e.g. missing links, it just has 3 buttons.
Not necessary for this PR, but I think it could be an improvement to make this example more elaborate.

Reply:
I'm gonna fix up a could of other things so will add that one in as well 👍